types/prosemirror-model/index.d.ts

```diff
@@ -1409,7 +1420,7 @@ export interface DOMOutputSpecArray {
     8?: DOMOutputSpec | 0 | undefined;
     9?: DOMOutputSpec | 0 | undefined;
 }
-export type DOMOutputSpec = string | Node | DOMOutputSpecArray | {dom: Node, contentDOM?: Node | undefined};
+export type DOMOutputSpec = string | Node | DOMOutputSpecArray | { dom: Node; contentDOM?: Node | undefined };
```

Comment:
Was this semicolon here inserted by accident? Other than that, looks good to me.

Reply:
This change is intentional. I used Prettier to format this file, according to [DefinitelyTyped's request](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes). TypeScript accepts both `,` and `;` here, but Prettier prefers `;`.

Reply:
Huh, I didn't know that. Nifty 👍